editor/inserter/menu.js

```diff
@@ -57,6 +72,10 @@ class InserterMenu extends Component {
 		} );
 	}
 
+	speakAssertive( message ) {
```

Comment:
Is this needed as a method of this class? In other words, is there a need for `this`? If not, it may be better to move it elsewhere. (const/function/annonymous in `constructor`).